docs/defi/chain-key-tokens/ckbtc/overview.mdx

```diff
@@ -55,14 +55,17 @@ The **ckBTC minter** is responsible for minting and burning ckBTC tokens. It use
 
 - For Bitcoin retrieval requests, the ckBTC minter burns ckBTC before transferring the corresponding BTC amount (minus fees) using a regular Bitcoin transaction.
 
-A detailed description of the ckBTC minter can be found in its [GitHub repository](https://github.com/dfinity/ic/tree/master/rs/bitcoin/ckbtc/minter).
+The ckBTC minter exposes a dashboard where you can find all the information about the minter's current state, the available UTXOs, outgoing transactions, current parameters, and the logs:
+
+- [ckBTC minter dashboard](https://mqygn-kiaaa-aaaar-qaadq-cai.raw.icp0.io/dashboard)
+- [ckTESTBTC minter dashboard](https://ml52i-qqaaa-aaaar-qaaba-cai.raw.icp0.io/dashboard)
+
+A detailed description of ckBTC can be found in the [Wiki](https://wiki.internetcomputer.org/wiki/Chain-key_Bitcoin).
 
 A simplified overview of the process to mint and transfer ckBTC is depicted in the following figure.
 
 ![ckBTC overview](../../_attachments/ckbtc-overview.png)
```

Comment:
the overview here should be updated as its missing the step where the user transfer bitcoin to the deposit address

Reply:
i personally think the bitcoin checker doesn't need to be surfaced, but depositing btc is very crucial for this flow to work so i'd definitely add it to the sequence diagram